tests/rados/test_omap_entries.py

```diff
@@ -92,19 +107,36 @@ def run(ceph_cluster, **kw):
                     f"All the OSD's from the acting set {acting_set} were restarted "
                     f"and object movement completed for pool {pool_name}"
                 )
+        # Verification of the Bug#2249003
+        try:
+            log.info(f"The omap object name is -{omap_obj}")
+            client_node.exec_command(cmd=cmd_clearomap, sudo=True)
+            cmd_clearomap = f"rados clearomap -p {crash_pool_name} {omap_obj}"
+            client_node.exec_command(cmd=cmd_clearomap, sudo=True)
         except Exception as e:
-            log.error(f"Failed with exception: {e.__doc__}")
-            log.exception(e)
-            return 1
-        finally:
-            log.info(
-                "\n \n ************** Execution of finally block begins here *************** \n \n"
+            log.info(f"Exception hit to execute but this is expected; {e}")
+        pass
```

Comment:
@SrinivasaBharath What is the purpose of `pass` here?

Reply:
There is no use with this pass. I removed it.